packages/select/src/components/select/multiSelect.tsx

```diff
@@ -56,6 +56,11 @@ export interface IMultiSelectState<T> {
 export class MultiSelect<T> extends React.PureComponent<IMultiSelectProps<T>, IMultiSelectState<T>> {
     public static displayName = "Blueprint2.MultiSelect";
 
+    public static defaultProps = {
```

Comment:
can we type this? `public static defaultProps: IMultiSelectProps<T> = { ... };`

what about defaults for the other props from the `IMultiSelectProps<T>` interface?

Reply:
unfortunately no, can't use `<T>` in static members. could do `Partial<IMultiSelectProps<any>>`. don't think there are others to default.

Reply:
roger on the typings. maybe `Partial<IMultiSelectProps<any>>` would still be nice then!

re: other defaults, I'm seeing a bunch of `popoverProps = {}`, and `tagInputProps = {}` in methods, isn't that defaultable?

Reply:
`popoverProps` and `tagInputProps` get default empty objects as ES6 default argument everywhere (` const { popoverProps = {} } = this.props`)

Reply:
right, but remind me why these can't be defaulted in `defaultProps` again?

Reply:
Actually they can. Should I add them to `defaultProps` and remove ES6 defaults?

Reply:
yeah, that'd be neat 👍

Reply:
@llorca We can't remove ES6 default params because TS throws `Object is possibly 'undefined'` otherwise. Adding `tagInputProps`, `popoverProps`, `selectedItems` to `defaultProps` will give no effect. Do you still want me to do this?

Reply:
ha snap. let's flag @giladgray, he'll know best